content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -308,28 +308,29 @@ If expanding underlying storage fails, the cluster administrator can manually re
 
 ## Types of Persistent Volumes
 
-PersistentVolume types are implemented as plugins.  Kubernetes currently supports the following plugins:
-
-* GCEPersistentDisk
-* AWSElasticBlockStore
-* AzureFile
-* AzureDisk
-* CSI
-* FC (Fibre Channel)
-* FlexVolume
-* Flocker
-* NFS
-* iSCSI
-* RBD (Ceph Block Device)
-* CephFS
-* Cinder (OpenStack block storage)
-* Glusterfs
-* VsphereVolume
-* Quobyte Volumes
-* HostPath (Single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)
-* Portworx Volumes
-* ScaleIO Volumes
-* StorageOS
+PersistentVolume types are implemented as plugins. Kubernetes currently supports the following plugins:
+
+* [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
+* [`azureDisk`](/docs/concepts/sotrage/volumes/#azuredisk) - Azure Disk
+* [`azureFile`](/docs/concepts/storage/volumes/#azurefile) - Azure File
+* [`cephfs`](/docs/concepts/storage/volumes/#cephfs) - CephFS volume
+* [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
+* [`csi`](/docs/concepts/storage/volumes/#csi) - Container Storage Interface (CSI)
+* [`fc`](/docs/concepts/storage/volumes/#fc) - Fibre Channel (FC)
+* [`flexVolume`](/docs/concepts/storage/volumes/#flexVolume) - FlexVolume
+* [`flocker`](/docs/concepts/storage/volumes/#flocker) - Flocker storage
+* [`gcePersistentDisk`](/docs/concepts/storage/volumes/#gcepersistentdisk) - GCE Persistent Disk
+* [`glusterfs`](/docs/concepts/storage/volumes/#glusterfs) - Glusterfs volume
+* [`hostPath`](/docs/concepts/storage/volumes/#hostpath) - HostPath volume (for single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)
```

Comment:
OK to leave as-is, because this text is copied from the original. The PR did make me notice it though.
```suggestion
* [`hostPath`](/docs/concepts/storage/volumes/#hostpath) - HostPath volume (for single node testing only; local storage is not supported in any way and **does not work** in a multi-node cluster)
```

I have another concern: I am not sure that this list item fully informs readers about whether `hostPath` is a supported volume type.
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/self-hosting/ mentions using `hostPath` volume types. If this truly is not supported, let's file a bug against either that doc or against kubeadm.

Reply:
Maybe https://github.com/kubernetes/website/pull/25039 is relevant.

Reply:
the hostPath volume type is a fully supported and required in some cases:
https://kubernetes.io/docs/concepts/storage/volumes/#hostpath

i believe the wording prior to this PR is just outdated:
> HostPath (Single node testing only -- local storage is not supported in any way and WILL NOT WORK in a multi-node cluster)

#sig-storage should confirm.

Reply:
Right, the text comes from the source code.
I'd be in favor of removing the phrase (and capitalization): "WILL NOT WORK". The phrase could be reworded or possibly link to a caution statement or guidelines in the Volumes page?

Reply:
Great. Let's wait for some suggestions from sig-storage.